### PR TITLE
Test: Add functional test for PG Targeting video lookup size

### PR DESCRIPTION
**Pull request by marki1an**
Test for https://github.com/prebid/prebid-server-java/pull/2058

### CODE REVIEW

File: src/test/groovy/org/prebid/server/functional/tests/pg/TargetingSpec.groovy [hunk 1]
```diff
@@ -529,4 +529,31 @@ class TargetingSpec extends BasePgSpec {
         then: "PBS had PG auction"
         assert auctionResponse.ext?.debug?.pgmetrics?.matchedWholeTargeting?.size() == lineItemSize
     }
+
+    def "PBS should be able to match video size targeting taken from imp[].video sources by INTERSECTS matching function"() {
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/pg/TargetingSpec.groovy:533]: Only INTERSECTS?

File: src/test/groovy/org/prebid/server/functional/tests/pg/TargetingSpec.groovy [hunk 2]
```diff
@@ -529,4 +529,31 @@ class TargetingSpec extends BasePgSpec {
...
+        given: "Default video bid request"
+        def defaultLineItemSize = LineItemSize.defaultLineItemSize
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/pg/TargetingSpec.groovy:535]: `def defaultLineItemSize` -> `def lineItemSize`

File: src/test/groovy/org/prebid/server/functional/tests/pg/TargetingSpec.groovy [hunk 3]
```diff
@@ -529,4 +529,31 @@ class TargetingSpec extends BasePgSpec {
...
+        }
+
+        and: "Planner response"
+        def plansResponse = PlansResponse.getDefaultPlansResponse(bidRequest.site.publisher.id).tap {
+            lineItems[0].targeting = new Targeting.Builder().addTargeting(AD_UNIT_SIZE, INTERSECTS, [defaultLineItemSize])
+                                                            .addTargeting(AD_UNIT_MEDIA_TYPE, INTERSECTS, [VIDEO])
+                                                            .build()
+        }
+        generalPlanner.initPlansResponse(plansResponse)
+        def lineItemSize = plansResponse.lineItems.size()
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/pg/TargetingSpec.groovy:548]: You don't need this variable. Please inline it.

**Review by Net-burst — CHANGES_REQUESTED** [review]

File: src/test/groovy/org/prebid/server/functional/tests/pg/TargetingSpec.groovy (hunk 1 above)
Comment (marki1an) [reply src/test/groovy/org/prebid/server/functional/tests/pg/TargetingSpec.groovy:533]: Yep

**Review by Net-burst — APPROVED** [review]
LGTM

**Pull request merged**